viewport/viewport.go

```diff
@@ -207,6 +207,11 @@ func (m *Model) LineUp(n int) (lines []string) {
 	return m.visibleLines()
 }
 
+// Lines returns all of the lines (both hidden and visible) within the viewport.
+func (m *Model) Lines() []string {
```

Comment:
@muesli: So we usually copy the type for getters. In the case of Bubble Tea, do you think it makes sense to instead start using pointer methods on Bubbles as an optimization?